apps/mobile/locales/en.json

```diff
@@ -303,7 +306,7 @@
   "Welcome to": "Welcome to",
   "Welcome to Hylo": "Welcome to Hylo",
   "Were almost done, are you sure you want to cancel signing-up?": "We're almost done, are you sure you want to cancel signing-up?",
-  "Were happy youre here": "We’re happy you’re here!",
+  "Were happy youre here": "We're happy you're here!",
```

Comment:
odd that the key doesn't have the single quotes

Reply:
Generally I avoid punctuation in keys because
1) I'm using i18n-ally
2) There are some special characters that are actually parsed by `i18next` as having a special meaning. 

So generally Its best to avoid them in the keys. I have been meaning to write up a i18n-ally setup in the readme...